manifests/params_agent.pp

```diff
@@ -224,6 +224,12 @@
       $ossec_syscheck_nodiff = '/etc/ssl/private.key'
       $ossec_syscheck_skip_nfs = 'yes'
 
+      # Audit
+      $audit_manage_rules                = false
+      $audit_buffer_bytes                = "8192"
+      $audit_backlog_wait_time           = "0"
+      $audit_rules                       = []
```

Comment:
I would prefer to define the default rules you put in the template into this array, audit_rules.

I already changed it in 5972eee

Thanks,